### PR TITLE
Include API metadata in output

### PR DESCRIPTION
**Pull request by osouthwell-scottlogic**
Adds new handlebar file to include api metadata in output

### CODE REVIEW

File: template/info.ts.handlebars [hunk 1]
```diff
@@ -0,0 +1,28 @@
+export interface infoInterface {
```
Comment (ColinEberhardt) [inline template/info.ts.handlebars:1]: Does this interface have to be generated? couldn't it be static, based on the OpenAPI spec?

**Review by jhowlett-scottlogic — APPROVED** [review]
You have addressed Colin's requested changes. Looks good to me

File: template/info.ts.handlebars [hunk 2]
```diff
@@ -0,0 +1,28 @@
+export interface InfoInterface {
+  title: string;
+  description?: string;
+  termsOfService?: string;
+  contact?: {
+    email?: string;
+  };
+  license?: {
+    name?: string;
```
Comment (jhowlett-scottlogic) [inline template/info.ts.handlebars:9]: info.licence.name is a required field

https://swagger.io/specification/#license-object

**Review by jhowlett-scottlogic — CHANGES_REQUESTED** [review]

**Review by jhowlett-scottlogic — APPROVED** [review]
Looks good to me

**Pull request merged**